src/table/utils/__tests__/handle-accessiblity.spec.ts

```diff
@@ -417,4 +417,34 @@ describe('handle-accessibility', () => {
       expect(announce).toHaveBeenCalledWith({ keys: ['SNTable.SelectionLabel.NotSelectedValue'] });
     });
   });
+
+  describe('handleKeyToCopyCell: ', () => {
+    let cellValue: Cell;
+    beforeEach(() => {
+      cellValue = {
+        qText: '251',
+      } as Cell;
+      console.log = jest.fn();
+    });
+
+    it('should copying value to clipboard successfully', () => {
+      Object.assign(navigator, {
+        clipboard: {
+          writeText: jest.fn().mockReturnValueOnce(Promise.resolve()),
+        },
+      });
+      handleAccessibility.handleKeyToCopyCell(cellValue);
+      expect(console.log).not.toHaveBeenCalled();
+    });
+
+    it('should fail copying value to clipboard', () => {
+      Object.defineProperty(navigator, 'clipboard', {
+        value: {
+          writeText: !jest.fn().mockReturnValueOnce(Promise.resolve()),
```

Comment:
Hmm actually it seems a bit trickier that that. Feel free to comment this test out, since we don't know what we want to do for the error handling anyway